app/modules/sightings/resources.py

```diff
@@ -428,7 +455,29 @@ def patch(self, args, sighting):
             )
 
         if edm_count > 0:
+            cleanup = SightingCleanup()
```

Comment:
This function is now over 100 lines long and while there may be worse examples in old world, I'm sure this is a pattern we don't want to follow. Have a think about how to break it up into more manageable chunks